src/devices/Display/LedDriver/category.txt

```diff
@@ -0,0 +1,2 @@
+display
+led driver
```

Comment:
You can pick only from here: https://github.com/dotnet/iot/blob/master/tools/device-listing/device-listing.cs#L55-L96

display
segment

match for sure. If this is LED driver as well then the original version (with `led`) was correct